ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -30,6 +44,11 @@ import {
 } from './components/ExceptionRequestTableCells';
 
 import './ExceptionRequestDetailsPage.css';
+import { getEntityPagePath } from '../WorkloadCves/searchUtils';
+import SeverityCountLabels from '../WorkloadCves/components/SeverityCountLabels';
+import CvssTd from '../WorkloadCves/components/CvssTd';
+import { VulnerabilitySeverityLabel } from '../WorkloadCves/types';
+import DateDistanceTd from '../WorkloadCves/components/DatePhraseTd';
```

Comment:
I could technically, try to move these up in the directory level to be outside both `WorkloadCVEs` and `ExceptionManagement`. Might make more sense that way.

Reply:
What do you think about moving the entire `/components/` directory up a level? I think there are things in there that will be used in the upcoming Node and Platform CVE pages, but I haven't looked closely enough to know how many of those components are reusable.

If the majority are really reusable it might be cleaner to keep them all in one general component directory for "Vulnerabilities" as a whole.

Reply:
Well, I wouldn't be opposed to that. It definitely would be cleaner to keep it all up there rather than partially moving some things around. Maybe we can do that. I'll see if I can refactor that after this PR too